README.md

```diff
@@ -75,6 +75,37 @@ We recommend developers use Seq locally to help ensure the audit event developme
 
 ![Sample Seq Output](./assets/seq-events-view.png)
 
+## Using custom sink for large audit events
+
+With increasing size in audit events large audit events couldn't be processed by `Kmd.Logic.Audit.Client.SerilogAzureEventHubs.SerilogAzureEventHubsAuditClient` implementation. As a solution to this we came up with a custom sink which based on the size of audit event will push the event to blob for large messages and keep the blob url in event hub. The default bytes limit is 256 KB which is customizable.
```

Comment:
"_will push the event to **blob** for large messages_". We probably need to specify that this is the blob of the customer which they need to create if they don't already have.

Reply:
Added a line as ` The blob will be owned by the user who is going to use the audit client`. Will it be meaningful?